app/views/home/_navbar.html.erb

```diff
@@ -1,17 +1,28 @@
 <% if current_user %>
-    <nav>
-        <div>
-            <% if request.path == root_path%>
-                <h2>Wallet Aid</h2>
-            <% else %>
-                <li><%= link_to '&#8249;'.html_safe, root_path %></li>
-            <% end %>
-        </div>
+  <nav>
+    <div>
+      <% if request.path == root_path %>
+        <h2>Wallet Aid</h2>
+      <% elsif request.path == new_category_path %>
+        <li><%= link_to '&#8249;'.html_safe, root_path %></li>
+      <% elsif request.path == new_category_expense_path %>
```

Comment:
- [ ] Good job by implementing the requested changes but now when I try to create a category I get an error, please kindly fix that 🙏🏾 🙏🏾 

<img width="771" alt="Screenshot 2023-10-17 at 17 45 02" src="https://github.com/BenM4k/rails_capstone/assets/74501523/4f90d702-8682-48cf-bbe2-ce53f8552fee">